docs/autotraining.rst

```diff
@@ -0,0 +1,41 @@
+autotraining
+============
+
+.. dfhack-tool::
+    :summary: Assigns citizens to a military squad until they have fulfilled their need for Martial Training
+    :tags: fort auto bugfix units
+
+Automation script for citizens to hit the gym when they yearn for the gains. Also passively builds military skills and physical stats.
```

Comment:
Oops, it looks like I accidentally removed the most important comment regarding language: I don't think that the whole "gym" "yearn" "gains" terminology fits with the way that documentation is usually phrased.

```suggestion
This script automatically assigns citizens with the need for military training to designated training squads.
```